### PR TITLE
Rolled back and only added blog changes

### PR DESCRIPTION
**Pull request by mmurray22**
Added a blog!

### CODE REVIEW

File: portfolio/src/main/webapp/blog.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
+    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <title>MM Portfolio</title>
+    <link rel="stylesheet" href="blog_style.css">
+    <script src="blog_script.js"></script>
```
Comment (ccondit) [inline portfolio/src/main/webapp/blog.html]: I still think you'll need the leading slashes on the css and js resources here in order for the container to render the style correctly. Have you tried invoking `mvn package appengine:run` to verify this?

**Review by ccondit — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/blog.html (hunk 1 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/blog.html]: Yes I have, but I can definitely add the ./ to for style purposes!

Comment (ccondit) [reply portfolio/src/main/webapp/blog.html]: Huh - when I tried this yesterday with Bianca's repo it wasn't rendering when running in the container - but indeed I am incorrect :). Just tried it for my own sanity and it seems to work either way. I would revert that change.

**Review by ccondit — APPROVED** [review]
LGTM after your revert my erroneous earlier suggestion. 

And thanks for breaking this out into a separate PR - made it much easier to review 😊

**Pull request merged**